nucleus/validate/client.py

```diff
@@ -78,6 +83,30 @@ def create_scenario_test(
                 "Must pass an evaluation_criteria to the scenario test! I.e. "
                 "evaluation_criteria = [client.validate.eval_functions.bbox_iou() > 0.5]"
             )
+        incorrect_type = [
+            crit
+            for crit in evaluation_criteria
+            if not isinstance(crit, EvaluationCriterion)
+        ]
```

Comment:
Might be more readable using `all`
```suggestion
        incorrect_type = all([isinstance(crit, EvaluationCriterion) for crit in evaluation_criteria])
```

Reply:
I'm actually using implicit list length check, an empty list evaluates to `False` and then using the `incorrect_type` as a list for creating a better error message.

Reply:
Renamed to `incorrect_types` and made a length check to make it look more list-y.